packages/flutter_localizations/lib/src/l10n/README.md

```diff
@@ -186,6 +186,19 @@ dart dev/tools/localizations/gen_localizations.dart --overwrite
 ```
 
 
+### Special handling for the Kannada (kn) translations
+
+The cupertino_kn.arb and material_kn.arb files contain unicode characters which
```

Comment:
I'm wondering if the wording here can be improved a ltitle, since the rewritten versions of the *_kn.arb files are what developers will see and this line might be confusing.
```suggestion
Originally, the cupertino_kn.arb and material_kn.arb files contain unicode characters which
```

Reply:
Sounds good, I made this update and a few related ones.